File: Łatwe/Java/Wiek segmentolka/WiekSegmentolka.java
import java.util.Scanner;

class WiekSegmentolka {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i = 0; i < t; i++) {
            int n = in.nextInt();
            int lifeSpan = n - 1;
            for (int j = 0; j < n; j++) {
                int numberOfLegs = in.nextInt();
                lifeSpan += numberOfLegs;
            }
            System.out.println(lifeSpan);
        }
    }
}

File: Łatwe/C++/Wiek segmentolka/WiekSegmentolka.cpp
#include <iostream>

int main() {
    int t;
    std::cin >> t;
    for (int i = 0; i < t; i++) {
        int n;
        std::cin >> n;
        int lifeSpan = n - 1;
        int numberOfLegs;
        for (int j = 0; j < n; j++) {
            std::cin >> numberOfLegs;
            lifeSpan += numberOfLegs;
        }
        std::cout << lifeSpan << std::endl;
    }
}
